docs/bigquery-usage.rst

```diff
@@ -455,11 +448,12 @@ located on Google Cloud Storage.  First, create the job locally:
    >>> from gcloud import bigquery
    >>> client = bigquery.Client()
    >>> table = dataset.table(name='person_ages')
-   >>> job = table.export_to_storage(bucket_name='bucket-name',
-   ...                               object_name_glob='export-prefix*.csv',
-   ...                               destination_format='CSV',
-   ...                               print_header=1,
-   ...                               write_disposition='truncate')
+   >>> job = client.extract_table_to_storage(
+   ...          'extract-person-ages', table,
+   ...          'gs://bucket-name/export-prefix*.csv')
```

Comment:
This is a "load from Cloud Storage" method:  it must take only a bucket/object(w/ optional glob).  Switching to URIs allows passing more than one (a back-end feature).